src/System.Private.CoreLib/shared/System/Threading/AsyncLocal.cs

```diff
@@ -72,8 +73,8 @@ public T Value
         void IAsyncLocal.OnValueChanged(object previousValueObj, object currentValueObj, bool contextChanged)
```

Comment:
`previousValueObj` and `currentValueObj` are compared to null should they be nullable? If so, should we also change IAsyncLocal interface?

Reply:
Yup, fixed. Thanks.